### PR TITLE
Print errors handling Prometheus disk and snapshot

### PR DESCRIPTION
**Pull request by jkaniuk**
Currently some error messages are silently ignored.

/cc wojtek-t

### CODE REVIEW

File: clusterloader2/pkg/prometheus/experimental.go [hunk 1]
```diff
@@ -128,6 +128,7 @@ func (pc *Controller) snapshotPrometheusDiskIfEnabled() error {
 		snapshotRetryTimeout,
 		func() (bool, error) {
 			err := pc.trySnapshotPrometheusDisk(pc.diskMetadata.name, snapshotName, pc.diskMetadata.zone)
+			klog.Error(err)
```
Comment (wojtek-t) [inline clusterloader2/pkg/prometheus/experimental.go:131]: I would prefer sth like:
```
if err != nil {
  klog.Errorf("Trying to snapshot prometheus disk failed: %v", err)
}
```

it's more informative

Comment (jkaniuk) [reply clusterloader2/pkg/prometheus/experimental.go]: Done

**Pull request merged**